### PR TITLE
docs: add contributing page

### LINKED ISSUE #1583: Docs: Add page for contributing
Add a page to the website about contributing that's easier to understand than sifting through a markdown file.

Should include:

- [ ] Complete setup instructions
- [ ] Contributing docs vs code
- [ ] Where to look in the codebase
- [ ] Instructions for submitting a PR
- [ ] Frequently Asked Questions

### PR DESCRIPTION
**Pull request by wesleygrimes**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #1583 

## What is the new behavior?

Adds new Contributing page to the NgRx.io website.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/marketing/contributing.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+NgRx is a community-maintained project that thrives due to the contributions of fellow developers. Whether it be documentation, issues, features, or tests, all contributions help this project in a meaningful way. This page serves as a "Getting Started Guide" on how to contribute to NgRx in different areas. If you also want to donate or sponsor this project, visit our [Open Collective](https://opencollective.com/ngrx) page.
+
+## Prerequisites
+
+- `yarn` - The NgRx library utilizes the `yarn` CLI tool. Please make sure that you have the latest stable release of `yarn` installed. For more information, visit the [Yarn Install Docs](https://yarnpkg.com/en/docs/install)
+
+## Contributing to the Docs and NgRx.io Website
+
+### Folder Structure
+Source code for the NgRx docs and the ngrx.io website exist under the `projects/ngrx.io/content` folder. If you are planning to contribute features and/or bug fixes relating to the docs or the website, chances are the code will be found there. The structure is split up between the following subfolders and key files:
```
Comment (brandonroberts) [inline projects/ngrx.io/content/marketing/contributing.md]:
```suggestion
Source code for the NgRx docs and the ngrx.io website exist under the `projects/ngrx.io/content` folder. If you are planning to contribute features and/or bug fixes relating to the docs or the website, the code is there. The structure is split up between the following subfolders and key files:
```

File: projects/ngrx.io/content/marketing/contributing.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+yarn serve-and-sync
+```
+
+After running this command, an instance of the website and docs will be available at `http://localhost:4200`
+
+## Contributing to the Library Modules
+
+### Folder Structure
+
+Source code for the NgRx library exists under the `modules/` project folder. If you are planning to contribute features and/or bug fixes relating to the library, chances are the code will be found in one of the `modules` sub-folders. These sub-folders mirror, for the most part, the `npm` packages for `@ngrx/*`:
```
Comment (brandonroberts) [inline projects/ngrx.io/content/marketing/contributing.md]:
```suggestion
Source code for the NgRx library exists under the `modules/` project folder. If you are planning to contribute features and/or bug fixes relating to the library, the code is in one of the `modules` sub-folders. These sub-folders mirror, for the most part, the `npm` packages for `@ngrx/*`:
```

File: projects/ngrx.io/content/marketing/contributing.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+```sh
+yarn test
+```
+
+## Submitting Pull Requests
+
+**Please follow these basic steps to simplify pull request reviews. If you don't you'll probably just be asked to anyway.**
+
+- Please rebase your branch against the current master.
+- Run the `Setup` command to make sure your development dependencies are up-to-date.
```
Comment (brandonroberts) [inline projects/ngrx.io/content/marketing/contributing.md]:
```suggestion
- Run the `Setup` instructions to make sure your development dependencies are up-to-date.
```

File: projects/ngrx.io/content/marketing/contributing.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+- We value keeping the API surface small and concise, which factors into whether new features are accepted.
+- Submit an issue with the prefix `RFC:` with your feature request.
+- The feature will be discussed and considered.
+- Once the PR is submitted, it will be reviewed and merged once approved.
+
+## <a name="commit"></a> Commit Message Guidelines
+
+We have very precise rules over how our git commit messages can be formatted. This leads to **more
+readable messages** that are easy to follow when looking through the **project history**. But also,
+we use the git commit messages to **generate the NgRx change log**.
```
Comment (brandonroberts) [inline projects/ngrx.io/content/marketing/contributing.md]:
```suggestion
we use the git commit messages to **generate the NgRx changelog**.
```

File: projects/ngrx.io/content/marketing/contributing.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+
+Any line of the commit message cannot be longer 100 characters! This allows the message to be easier
+to read on GitHub as well as in various git tools.
+
+The footer should contain a [closing reference to an issue](https://help.github.com/articles/closing-issues-via-commit-messages/) if any.
+
+Samples: (even more [samples](https://github.com/ngrx/platform/commits/master))
+
+```
+docs(changelog): update changelog to beta.5
```
Comment (brandonroberts) [inline projects/ngrx.io/content/marketing/contributing.md]:
```suggestion
docs: add new example to selectors guide
```

File: projects/ngrx.io/content/marketing/contributing.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+```
+
+```
+fix(release): need to depend on latest rxjs and zone.js
```
Comment (brandonroberts) [inline projects/ngrx.io/content/marketing/contributing.md]:
```suggestion
fix(store): adjust mock store to handle selectors with props
```

File: projects/ngrx.io/content/marketing/contributing.md [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+
+The version in our package.json gets copied to the one we publish, and users need the latest of these.
```
Comment (brandonroberts) [inline projects/ngrx.io/content/marketing/contributing.md]:
```suggestion
```

File: projects/ngrx.io/content/marketing/contributing.md [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+<footer>
+```
+
+The **header** is mandatory and the **scope** of the header is optional.
+
+Any line of the commit message cannot be longer 100 characters! This allows the message to be easier
+to read on GitHub as well as in various git tools.
+
+The footer should contain a [closing reference to an issue](https://help.github.com/articles/closing-issues-via-commit-messages/) if any.
+
```
Comment (brandonroberts) [inline projects/ngrx.io/content/marketing/contributing.md]:
```suggestion
```
feat(router-store): add routerState config option

Closes #1834
```
```

File: projects/ngrx.io/content/marketing/contributing.md [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+- `projects/ngrx.io/content/examples` - Code examples referenced on the website and in the docs should be placed under this folder
+- `projects/ngrx.io/content/guide` - The majority of official NgRx documentation exists under this folder. There are sub-folders for each module.
+- `projects/ngrx.io/content/marketing` - The majority of the additional website pages such as resources, events, about the NgRx team exist under this folder.
+- `projects/ngrx/io/content/events.json` - The Events page dynamically displays two tables, one for Past Events and one for Present events based on the contents of this file.
+- `projects/ngrx/io/content/resources.json` - The Resources page dynamically displays a list of NgRx resources based on the contents of this file.
+- `projects/ngrx/io/content/navigation.json` - All of the navigation and menus on the website and docs are populated from this JSON file. If a new page is being added or adjusted make sure to update this file as well.
+
+### Submission Guidelines
+
+The NgRx team strives for excellence when it comes to documentation.
```
Comment (brandonroberts) [inline projects/ngrx.io/content/marketing/contributing.md]:
```suggestion
The NgRx team values quality documentation as a way to help new and existing users understand and use features of the platform. The guidelines below help to ensure that quality for our users.
```

File: projects/ngrx.io/content/marketing/contributing.md (hunk 8 above)
Comment (wesleygrimes) [reply projects/ngrx.io/content/marketing/contributing.md]: I believe I have updated this one as requested, can you confirm?

Comment (brandonroberts) [reply projects/ngrx.io/content/marketing/contributing.md]: 👍

**Pull request merged**